app/src/main/java/com/ncnf/feed/ui/FeedFragment.java

```diff
@@ -112,4 +161,33 @@ public boolean onOptionsItemSelected(@NonNull MenuItem item) {
         }
         return true;
     }
+
+    private void destroyChildFragment(FragmentManager fragmentManager, Fragment fragment, ConstraintLayout feedContainer, OnBackPressedCallback callback){
+        fragmentManager.beginTransaction().remove(fragment).commit();
+        feedContainer.setVisibility(View.GONE);
+        recycler.setVisibility(View.VISIBLE);
+        callback.setEnabled(false);
+    }
+
+    private void actualise_events(){
+        final List<SocialObject> result = new ArrayList<>();
+
+        CompletableFuture<List<SocialObject>> completableFuture = new DatabaseService().eventGeoQuery(Settings.getUserPosition(), Settings.getCurrentMaxDistance() * 1000);
```

Comment:
Shouldn't you inject the database service ? How do you mock in your tests ?

Reply:
Well I didn't and everything went fine... Also on Cirrus too, those same tests pass...